onnx/version_converter/adapters/gridsample_19_20.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) ONNX Project Contributors
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Adapter for GridSample in default domain from version 19 to 20
+
+#pragma once
+
+namespace ONNX_NAMESPACE {
+namespace version_conversion {
+
+class GridSample_19_20 final : public Adapter {
+ public:
+  explicit GridSample_19_20() : Adapter("GridSample", OpSetID(19), OpSetID(20)) {}
+
+  void adapt_gridsample_19_20(std::shared_ptr<Graph>, Node* node) const {
+    if (node->s(kmode) == "bilinear") {
```

Comment:
I think this may need to be `if (node->hasAttribute(kmode) && (node->s(kmode) == "bilinear"))`. Same below.

Reply:
Fixed.